JitsiMeetJS.js

```diff
@@ -69,28 +63,29 @@ function getLowerResolution(resolution) {
 }
 
 /**
- * Checks the available devices in options and concatenate the data to the
- * name, which will be used as analytics event name. Adds resolution for the
- * devices.
- * @param name name of event
- * @param options gum options
- * @returns {*}
+ * Extracts from an 'options' objects with a specific format
+ * (TODO what IS the format?) the attributes which are to be logged in analytics
+ * events.
+ *
+ * @param options gum options (???)
+ * @returns {*} the attributes to attach to analytics events.
  */
-function addDeviceTypeToAnalyticsEvent(name, options) {
-    let ret = name;
+function getAnalyticsAttributesFromOptions(options) {
+    const attributes = {
+        audioRequested:
+            options.devices.indexOf('audio') !== -1,
+        videoRequested:
+            options.devices.indexOf('audio') !== -1,
```

Comment:
@bgrozev, why do both `audioRequested` and `videoRequested` get assigned one and the same value `options.devices.indexOf('audio') !== -1`? Maybe `videoRequested` should use `'video'`?

Reply:
Yes, good catch!